cdap-e2e-tests/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -3,3 +3,8 @@ clientUrl=http://localhost:11011
 serverUrl=https://placeholder.com/api
 # command to generate token: gcloud auth print-access-token
 serverAccessToken=placeholder
+expectedPluginName=Anaplan
```

Comment:
And the start and end comments section for HUB related properties as this is common file. Refer google cloud repo for same.

Reply:
added.